helm/extendedapi/templates/deployment.yaml

```diff
@@ -18,6 +18,8 @@ spec:
     spec:
       nodeSelector:
         dominodatalab.com/node-pool: platform
+      serviceAccountName: operator-webhook
+      automountServiceAccountToken: true  

```

Comment:
I think we should create a separate service account here. The `operator-webhook` has too many permissions. Are we deploying in the platform-namespace? We should deploy in the field namespace. 

If we are fetching passwords directly using the api, there is no need to put this in the platform namespace